server/src/http/feature_refresher.rs

```diff
@@ -154,6 +158,7 @@ impl FeatureRefresher {
             engine_cache: Arc::new(Default::default()),
             refresh_interval: chrono::Duration::seconds(10),
             persistence: None,
+            open: Default::default(),
```

Comment:
Me again with my love of explicit stuff

```suggestion
            open: false,
```